### PR TITLE
The loop of any circle is nontrivial

### PR DESCRIPTION
**Pull request by EgbertRijke**
This PR shows that the loop on any circle is nontrivial. I also changed "fundamental cover" to "universal cover" because that seemed to be the conventional name.

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
Yay!

**Comment by EgbertRijke** [comment]
Lol this is not a milestone :P

File: src/synthetic-homotopy-theory/universal-cover-circle.lagda.md [hunk 1]
```diff
@@ -386,241 +396,265 @@ equiv-dependent-identification-contraction-total-space'
...
+### The loop of the circle is nontrivial
+
+```agda
+module _
+  {l1 : Level} {X : UU l1} (l : free-loop X)
+  (H : {l2 : Level} → dependent-universal-property-circle l2 l)
+  where
+
+  is-nontrivial-loop-dependent-universal-property-circle :
+    ¬ (loop-free-loop l ＝ refl)
```
Comment (fredrik-bakke) [inline src/synthetic-homotopy-theory/universal-cover-circle.lagda.md:647]: Do you mind writing it like this?

```suggestion
    loop-free-loop ≠ refl
```

requires importing `foundation.negated-equality`

File: src/synthetic-homotopy-theory/universal-cover-circle.lagda.md [hunk 2]
```diff
@@ -386,241 +396,265 @@ equiv-dependent-identification-contraction-total-space'
...
-  ( equiv-fundamental-cover-circle l dup-circle (base-free-loop l))
+  ( inv-equiv (compute-fiber-universal-cover-circle l dup-circle)) ∘e
+  ( equiv-universal-cover-circle l dup-circle (base-free-loop l))
+```
+
+### The loop of the circle is nontrivial
+
+```agda
+module _
+  {l1 : Level} {X : UU l1} (l : free-loop X)
```
Comment (fredrik-bakke) [inline src/synthetic-homotopy-theory/universal-cover-circle.lagda.md:642]: I'm not a huge fan of using the name `l` for a loop, it's too easy to confuse with a universe level in my opinion. How about calling it `ω` or maybe `o`?

**Review by fredrik-bakke — APPROVED** [review]
Thanks for the help!

**Comment by fredrik-bakke** [comment]
I will merge this without having the comments resolved so that I can use the result. Thanks again! :)

**Pull request merged**